adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -968,6 +968,20 @@ private boolean isHiddenPaymentMethod(PaymentMethod paymentMethod) {
         return false;
     }
 
+    private List<StoredPaymentMethod> getStoredOneClickPaymentMethods(PaymentMethodsResponse response) {
+        List<StoredPaymentMethod> storedPaymentMethodList = null;
+        if (response.getOneClickPaymentMethods() != null && response.getStoredPaymentMethods() != null) {
+            Set<String> oneClickIds = response.getOneClickPaymentMethods().stream()
+                    .map(com.adyen.model.checkout.RecurringDetail::getRecurringDetailReference)
+                    .collect(Collectors.toSet());
+            storedPaymentMethodList = response.getStoredPaymentMethods().stream()
```

Comment:
minor improvement for a future ticket: use directly the OneClickPaymentMethods structure if possible

Reply:
Agreed. Just avoided doing that now cause it would require bigger changes on other classes and on the way we render the cards component.

Reply:
Let's create a separate story to address this. Approving for now.